<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMvBlobSortOutput</class>
 <widget class="QWidget" name="CMvBlobSortOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blob排序输出</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>382</width>
         <height>602</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>输入设置</string>
        </attribute>
        <widget class="QLabel" name="label_funcName">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>检测器名称：</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_enableFunc">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>20</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>启用检测器</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_funcName">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
         <property name="overwriteMode">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_input">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>341</width>
           <height>391</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>输入名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>输入来源</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Blob输入</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>某个检测器的输出</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>参数设置</string>
        </attribute>
        <widget class="QLabel" name="label_SortingCriteria">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>排序标准：</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_SortingCriteria">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>面积</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中心X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中心Y</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBox_SortType">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>50</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>原始</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>升序</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>降序</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_SortType">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>排序类型：</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_ones">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>单次</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_affirm">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbPrev_cancel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CMvContourMatching.qrc"/>
 </resources>
 <connections/>
</ui>
